src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.csproj

```diff
@@ -4,19 +4,13 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>Microsoft.AspNetCore.Server.IIS</PackageId>
     <Description>Provides support for hosting ASP.NET Core in IIS using the AspNetCoreModule.</Description>
-    <IsPackable Condition="'$(OS)' != 'Windows_NT'">false</IsPackable>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;iis</PackageTags>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PackNativeAssets Condition="'$(OS)' == 'Windows_NT'">True</PackNativeAssets>
```

Comment:
Nitpick: I think "true" is the standard xml truthy value, let's not confuse things by using the uppercase.